pkg/receiver/jobreceiver/builder/config.go

```diff
@@ -0,0 +1,69 @@
+package builder
```

Comment:
This package is basically the glue between Stanza's receiver adapter and our code.

NewOperatorBuilder gives stanza a way to build an input operator to use, hooks up one of our output handlers (event or log entries) to that input operator, and hooks up our main loop to that output handler through the consumer interface.